Adding cisco spark connector in opsdroid (#1041)
opsdroid/opsdroid: docs/connectors/webexteams.md

> # opsdroid connector webex teams

Reviewer: Can we update this to be consistent with the other docs pages?

```suggestion
# Webex Teams connector 
```

A connector for [opsdroid](https://github.com/opsdroid/opsdroid) to interact with [Webex Teams](https://www.webex.com/team-collaboration.html).

> _Note: Webex Teams was used to be called cisco spark. We changed cisco spark connector to webex teams connector 

Reviewer: You need to close the italics like this.

```suggestion
_Note: Webex Teams was used to be called cisco spark. We changed cisco spark connector to webex teams connector_ 
```

> ##### Note: Webex Teams was used to be called cisco spark. We changed cisco spark connector to webex teams connector 

Reviewer: Let's make this italic rather than a title
```suggestion
_Note: Webex Teams was used to be called cisco spark. We changed cisco spark connector to webex teams connector _
```

## Usage

**This connector requires that the opsdroid web server is internet facing.**

 - Visit https://developer.webex.com/ and log in
 - Go to "My Apps"
 - Click the plus button to create a new app and select "Create a Bot"
 - Fill in the details and click "Add Bot"
 - Scroll down and find your access token

## Configuration

```yaml
connectors:
  - name: webexteams
    # required

>     webhook-url: http(s)://<host>:<port>  # Url for Spark to connect to your bot

Reviewer: Missed rename here

```suggestion
    webhook-url: http(s)://<host>:<port>  # Url for Webex Teams to connect to your bot
```
